spec/controllers/idv/how_to_verify_controller_spec.rb

```diff
@@ -35,10 +35,36 @@
   end
 
   describe '#update' do
```

Comment:
i think it might be useful to add a spec for the undo_step

Reply:
Done. Resolved with commit fc6308d7804a87ec5c53f4b1eb1680960ec37e1c